docs/SECURITY.md

```diff
@@ -18,10 +18,17 @@ limitations under the License.
 
 The CSM services/repositories are inspected for security vulnerabilities via [gosec](https://github.com/securego/gosec).
 
-Every issue detected by `gosec` is mapped to a [CWE (Common Weakness Enumeration)](http://cwe.mitre.org/data/index.html) which describes in more generic terms the vulnerability.  The exact mapping can be found at https://github.com/securego/gosec in the issue.go file. The list of rules checked by `gosec` can be found [here](https://github.com/securego/gosec#available-rules).
+Every issue detected by `gosec` is mapped to a [CWE (Common Weakness Enumeration)](http://cwe.mitre.org/data/index.html) which describes in more generic terms the vulnerability.  The exact mapping can be found at https://github.com/securego/gosec in the `issue.go` file. The list of rules checked by `gosec` can be found [here](https://github.com/securego/gosec#available-rules).
```

Comment:
Please remove everything about the heading "Reporting Security Issues/Vulnerability"

Reply:
Done